shell/platform/darwin/ios/framework/Source/accessibility_bridge_test.mm

```diff
@@ -1664,4 +1664,51 @@ - (void)testFlutterSemanticsScrollViewManagedObjectLifecycleCorrectly {
   // flutterSemanticsScrollView) will cause an EXC_BAD_ACCESS.
   XCTAssertFalse([flutterSemanticsScrollView isAccessibilityElement]);
 }
+
+- (void)testPlatformViewDestructorDoesNotCallSemanticsAPIs {
+  class TestDelegate : public flutter::MockDelegate {
+   public:
+    void OnPlatformViewSetSemanticsEnabled(bool enabled) override { set_semantics_enabled_calls++; }
+    int set_semantics_enabled_calls = 0;
```

Comment:
nit: google c++ style says this should have a postfix `_` (only structs have fields without them).

I was thinking that you'd show it called through once so the final assert would be 1, not 2, not a huge deal.